packages/web-components/@sfx/sayt/CHANGELOG.md

```diff
@@ -8,3 +8,4 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - SFX-191: Added the `sayt` component.
   - This component renders `sayt` related components such as `sfx-autocomplete` and `sfx-products`.
+  -SFX-295: Added functionality to accommodate multiple `sayts`. 
```

Comment:
We don't need to add this line. Given that we're still in an unreleased state, we can largely ignore the changelog. If there is a major change that is not core to the idea of SAYT, it could be worth adding for clarity.